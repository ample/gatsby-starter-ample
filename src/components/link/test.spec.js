import React from "react"
import renderer from "react-test-renderer"

import { component as Link, fixtures } from "."

describe("Link", () => {
  it("renders internal link correctly", () => {
    const tree = renderer.create(<Link {...fixtures.internal} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it("renders external link correctly", () => {
    const tree = renderer.create(<Link {...fixtures.external} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
