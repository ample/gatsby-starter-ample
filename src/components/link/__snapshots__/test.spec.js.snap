// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders external link correctly 1`] = `
<a
  className="link is_external"
  href="https://ample.co"
  rel="noopener"
  target="_blank"
>
  And I'm external
</a>
`;

exports[`Link renders internal link correctly 1`] = `
<a
  className="link"
  href="/components/link"
>
  I'm an internal link
</a>
`;
